javaext/bundle.target

```diff
@@ -14,20 +14,31 @@
         <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
             <unit id="org.eclipse.m2e.feature.feature.group" version="0.0.0"/>
             <unit id="org.eclipse.m2e.logback.feature.feature.group" version="0.0.0"/>
-	        <repository location="https://download.eclipse.org/technology/m2e/snapshots/latest/"/>
+            <repository location="https://download.eclipse.org/technology/m2e/releases/2.0.1/"/>
+        </location>
+        <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+            <unit id="org.eclipse.equinox.core.feature.feature.group" version="0.0.0"/>
+            <unit id="org.eclipse.equinox.core.sdk.feature.group" version="0.0.0"/>
+            <unit id="org.eclipse.equinox.executable.feature.group" version="0.0.0"/>
+            <unit id="org.eclipse.equinox.p2.core.feature.source.feature.group" version="0.0.0"/>
+            <unit id="org.eclipse.equinox.sdk.feature.group" version="0.0.0"/>
+            <unit id="org.eclipse.jdt.source.feature.group" version="0.0.0"/>
+            <unit id="org.eclipse.sdk.feature.group" version="0.0.0"/>
+            <unit id="org.mockito.mockito-core" version="0.0.0"/>
+            <repository location="https://download.eclipse.org/eclipse/updates/4.25/"/>
         </location>
         <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
             <unit id="org.eclipse.xtend.sdk.feature.group" version="0.0.0"/>
             <unit id="org.eclipse.xtext.sdk.feature.group" version="0.0.0"/>
-            <repository location="https://download.eclipse.org/releases/2022-03/"/>
+            <repository location="https://download.eclipse.org/releases/2021-09/202109151000/"/>

```

Comment:
2022-03 is higher than 2021-09/202109151000, no need to downgrade.

Reply:
it seems that 2022-03 doesn't contain the above plugins:
```
Contains: Unable to locate installable unit org.eclipse.xtend.sdk.feature.group 0.0.0
Contains: Unable to locate installable unit org.eclipse.xtext.sdk.feature.group 0.0.0
```
 see also: https://github.com/eclipse/eclipse.jdt.ls/blob/dbd035c6aa036b6ee7dd7ea503de548f33fe641c/org.eclipse.jdt.ls.target/org.eclipse.jdt.ls.tp.target#L35

Reply:
ok.